Packs/MicrosoftTeams/Scripts/ConfigureAzureApplicationAccessPolicy/ConfigureAzureApplicationAccessPolicy.ps1

```diff
@@ -0,0 +1,40 @@
+Import-Module MicrosoftTeams
+
+function Main()
+{
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidUsingConvertToSecureStringWithPlainText', '')]
+    param()
+    $dargs = $demisto.Args()
+    $plain_username = $dargs.username
+    $plain_pass = $dargs.password
+    $password = (ConvertTo-SecureString -String $plain_pass -AsPlainText -Force)
+    $app_id = $dargs.app_id
+    $identity = $dargs.identity
+    try
+    {
+        $credential = New-Object System.Management.Automation.PSCredential($plain_username, $password)
+
+        Connect-MicrosoftTeams -Credential $credential
+
+        New-CsApplicationAccessPolicy -Identity Test-policy -AppIds $app_id
+        Grant-CsApplicationAccessPolicy -PolicyName Test-policy -Identity $identity
+
+        $human_readable = "Access policy was given"
+        $context = @{
+            "ConfigureAzureApplicationAccessPolicy" = @{
+                "Status" = "Access policy was given"
```

Comment:
If you're outputting the status, you shall enter here the username as well.

Reply:
I was thinking about it but then maybe I should add the app_id too?
There are both inputs to the script, wdut?

Reply:
Well, you need at least one identifier, so just pick on or both.